src/openrct2/ride/TrackDesign.cpp

```diff
@@ -184,8 +184,7 @@ static rct_track_td6 * track_design_open_from_td4(uint8 * src, size_t srcLength)
     }
 
     // Convert RCT1 vehicle type to RCT2 vehicle type. Intialise with an string consisting of 8 spaces (and no nul terminator).
-    rct_object_entry vehicleObject = {0x80, ""};
-    memset(vehicleObject.name, ' ', sizeof(vehicleObject.name));
+    rct_object_entry vehicleObject = {0x80, "        "};
```

Comment:
This will initialise `vehicleObject` with a string that _does_ include a null terminator, unlike what the comment above suggests.

Reply:
The comment is still true in a way: `name` won't contain a nul terminator. Anyway, I have adjusted the comment.